doc/src/manual/ranch.handshake.asciidoc

```diff
@@ -9,11 +9,12 @@ ranch:handshake - Perform the transport handshake
 [source,erlang]
 ----
 handshake(Ref)       -> handshake(Ref, [])
```

Comment:
Just noticed that it defaults Opts to an empty list. That is unfortunate since socket options are `any()`. Maybe we should have a `Transport:handshake/2` without the options so that the default options, if any, can be set directly from the `Transport` module.

Reply:
Hm, yeah, that's something that was nagging me ocassionally, but never got round to it. Same for `handshake_continue` then, btw.

Just mentioning, that would be sort of an API change, in the sense that custom transports will have to implement two additional callbacks. Do we care?

Reply:
Now's the time to do it.

Reply:
Ok, changed that, same place as in the above comment plus `ranch_transport`, `ranch_ssl`, `ranch_tcp`.